packages/grid/_modules_/grid/models/gridCellClass.ts

```diff
@@ -1,11 +1,11 @@
 import { GridCellParams } from './params/gridCellParams';
 
 /**
- * A function used to process cellClassParams.
+ * A function used to process cellClassName params.
  */
-export type GridCellClassFn = (params: GridCellParams) => string | string[];
+export type GridCellClassFn = (params: GridCellParams) => string;
```

Comment:
Why does it need to be public?

```suggestion
type GridCellClassFn = (params: GridCellParams) => string;
```

Reply:
why does it need to be private? If it is public, ppl can import it and type their function to make sure and avoid runtime errors

Reply:
They can already do it `GridColDef['cellClassName']`

Reply:
I guess that applies to all the other comments - I followed the pattern done for the `cellClassName`. If we go for the not exporting it option then I'll change it for both cases.

Reply:
It's not really important. I have raised the point as it seems to be a quick win. What draw my attention is that it's 1. inconsistent (we don't export a type for the each property of `GridColDef`, e.g. getValue ) 2. doesn't seem easy for a developer to discover (felt that the only way us to read the source, which developer's should have to). 3. the use case seems marginal with already an alternative.